configure-repo.sh

```diff
@@ -0,0 +1,40 @@
+#!/bin/sh
```

Comment:
```suggestion
#!/bin/bash
```

This got rid of the `[[` error I mention below